CRM/Report/Form/Contact/Relationship.php

```diff
@@ -670,18 +703,7 @@ public function alterDisplay(&$rows) {
           = $rows[$rowNum]['civicrm_contact_b_sort_name_b'] . ' (' .
           $rows[$rowNum]['civicrm_contact_b_id'] . ')';
         $rows[$rowNum]['civicrm_contact_b_sort_name_b_link'] = $url;
-        $rows[$rowNum]['civicrm_contact_b_sort_name_b_hover'] = ts("View Contact details for this contact.");
-        $entryFound = TRUE;
-      }
-
-      if (array_key_exists('civicrm_relationship_relationship_id', $row) &&
-        array_key_exists('civicrm_contact_id', $row)
-      ) {
-        $url = "/civicrm/contact/view/rel?reset=1&action=update&rtype=a_b&cid=" .
-          $row['civicrm_contact_id'] . "&id=" .
-          $row['civicrm_relationship_relationship_id'];
-        $rows[$rowNum]['civicrm_relationship_relationship_id_link'] = $url;
-        $rows[$rowNum]['civicrm_relationship_relationship_id_hover'] = ts("Edit this relationship.");
```

Comment:
This causes the regression for the requirement of https://issues.civicrm.org/jira/browse/CRM-14656. Any specific reason for removing the id link from the reports ?